### DISCUSSION TITLE
Cannot install DuckDB extension in a Kubernetes pod

### DISCUSSION THREAD

**Discussion by AmanSinghAS** [duckdb/duckdb#12294] [General]
I am trying to install httpfs extension in, and am succesfully able to do so locally, when I deploy to a dev environment in a kubernetes pod upon attempting to install the extension I run into:
IOException("IO Error: Can't find the home directory at '' Specify a home directory using the SET home_directory='/path/to/dir' option.")
Is there a common workaround?

Last mention of this I found is here: https://github.com/duckdb/duckdb/issues/3855

**Comment by Tishj** [comment]
It sounds like the error is explaining how to fix it
Also setting the HOME env var would probably fix it
